doc/radosgw/admin.rst

```diff
@@ -473,6 +473,157 @@ commands. ::
    be restarted for the changes to take effect.
 
 
+Rate Limit Management
```

Comment:
are these configuration per tenant or across tenants?

Reply:
per tenant

Reply:
so please add this information in the doc